packages/react-components/react-spinbutton/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -260,6 +278,39 @@ describe('SpinButton', () => {
     expect(onChange.mock.calls[1][1]).toEqual({ value: 2, displayValue: undefined });
   });
 
+  it('calls on change when defaultValue is `null` when uncontrolled', () => {
```

Comment:
> it('calls on change when defaultValue is `null` when uncontrolled' [](http://example.com/codeflow?start=1&length=67)

Consider potentially adding a tests the number pf onChange calls for multiple steps/blurs/ENTERs when the value is null (e.g. `expect(onChange).toHaveBeenCalledTimes(x);`) to explicitly keep track of how onChange is expected to get called in this case. I feel like if there's not a test like this, then the implicitly expected number of times onChange gets called in this case could change unexpectedly

(this is similar to my other comment in `useSpinButton`)